utils/build-script-impl

```diff
@@ -1625,7 +1561,7 @@ for deployment_target in "${NATIVE_TOOLS_DEPLOYMENT_TARGETS[@]}" "${CROSS_TOOLS_
 
                 build_targets=(all "${SWIFT_STDLIB_TARGETS[@]}")
                 if [[ $(true_false "${build_perf_testsuite_this_time}") == "TRUE" ]]; then
-                    native_swift_tools_path=$(build_directory macosx-x86_64 swift)/bin
+                    native_swift_tools_path=$(build_directory "${HOST_TARGET}" swift)/bin
```

Comment:
I also noticed this hard-coded path, so I amended the commit to remove it as well.